src/Writers/LaTeXWriter.jl

```diff
@@ -260,7 +260,7 @@ const LEXER = Set([
 ])
 
 function latex(io::IO, code::Markdown.Code)
-    language = if code.language == "jldoctest"
+    language = if first(split(code.language)) == "jldoctest"
```

Comment:
It seems it needs an `isempty` check here as well (HTML already has it). Praised be the automatic tests :laughing: